tests/sls/buildbot/master/config_lint.py

```diff
@@ -13,7 +13,7 @@ def run():
         'config'
     )
     # Have to specify master.cfg separately because it is not a .py file
-    command = ['flake8', CONF_DIR, os.path.join(CONF_DIR, 'master.cfg')]
+    command = ['flake8', CONF_DIR, "home/servo/buildbot/master/master.cfg"]
```

Comment:
Revert this change?

Reply:
No, this is intentional, this lets us lint the generated file.

The other option is to locally jinja it but I tried it and the salt jinja format is weird so I'd have to convert first.